db/replication-alarms.yml

```diff
@@ -0,0 +1,97 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Creates alarms related to ssh tunnel remote server -> rds db replication
+Parameters:
+  OpsWarnSnsTopic:
+    Default: ''
+    Description: ops warn SNS Topic arn
+    Type: String
+  EnvironmentType:
+    Type: String
+    AllowedValues:
+      - staging
+      - production
+  DatabaseName:
+    Default: 'mediajoint_production'
+    Description: Name of the database
+    Type: String
+  TunnelInstanceId:
+    Default: ''
+    Description: Instance id (i.e. i-something) of the EC2 server
+    Type: String
+Resources:
+  ReplicationIoHaltedAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[Replication][${EnvironmentType}][SlaveIORunning] Not running"
+      AlarmActions: [!Ref OpsWarnSnsTopic]
+      InsufficientDataActions: [!Ref OpsWarnSnsTopic]
+      OKActions: [!Ref OpsWarnSnsTopic]
+      AlarmDescription: IO replication not running
+      ComparisonOperator: LessThanThreshold
+      Dimensions:
+        - Name: Database
+          Value: !Ref DatabaseName
+        - Name: Environment
+          Value: !Ref EnvironmentType
+        - Name: InstanceID
+          Value: !Ref TunnelInstanceId
+      EvaluationPeriods: 2
+      MetricName: SlaveIORunning
+      Namespace: PRX/Data
+      Period: 60
+      Statistic: Average
+      Threshold: 1
+      TreatMissingData: notBreaching
+      Unit: Count
+  ReplicationSqlHaltedAlarm:
+    Type: "AWS::CloudWatch::Alarm"
+    Properties:
+      ActionsEnabled: true
+      AlarmName: !Sub "[Replication][${EnvironmentType}][SlaveSQLRunning] Not running"
+      AlarmActions: [!Ref OpsWarnSnsTopic]
+      InsufficientDataActions: [!Ref OpsWarnSnsTopic]
+      OKActions: [!Ref OpsWarnSnsTopic]
+      AlarmDescription: SQL replication not running
+      ComparisonOperator: LessThanThreshold
+      Dimensions:
+        - Name: Database
+          Value: !Ref DatabaseName
+        - Name: Environment
+          Value: !Ref EnvironmentType
+        - Name: InstanceID
+          Value: !Ref TunnelInstanceId
+      EvaluationPeriods: 2
+      MetricName: SlaveSQLRunning
+      Namespace: PRX/Data
+      Period: 60
+      Statistic: Average
```

Comment:
What does the value of this metric represent? A boolean?

Reply:
boolen, yes - this is a 1 or 0, a 1 if it is running, a 0 if it is not.